cardigan/stories/components/Button.js

```diff
@@ -17,6 +17,5 @@ const ButtonExample = () => {
   );
 };
 
-const stories = storiesOf('Components', module);
-stories
-  .add('Buttons: Button', ButtonExample, {info: Readme});
+const stories = storiesOf('Components/Buttons', module);
+stories.add('Button', ButtonExample, { info: Readme });
```

Comment:
So we are keeping the categorisations?

Reply:
I've kept them for now, but I'm not sure they're useful